main.go

```diff
@@ -1110,12 +1111,16 @@ func loadCachedOutputs() error {
 }
 
 func (tf *transformer) findReflectFunctions(files []*ast.File) {
+	seenReflectParams := make(map[*types.Var]bool)
 	visitFuncDecl := func(funcDecl *ast.FuncDecl) {
-		funcObj := tf.info.ObjectOf(funcDecl.Name).(*types.Func)
+		funcObj := tf.info.Defs[funcDecl.Name].(*types.Func)
 		funcType := funcObj.Type().(*types.Signature)
 		funcParams := funcType.Params()
 
-		seenReflectParams := make(map[*types.Var]bool)
+		// Clear and reuse the map.
+		for obj := range seenReflectParams {
+			delete(seenReflectParams, obj)
+		}
```

Comment:
Use https://pkg.go.dev/golang.org/x/exp/maps#Clear here

Reply:
neat, I wasn't aware of this shortcut! I guess both are optimized by the compiler equally.